tools/CMakeLists.txt

```diff
@@ -5,4 +5,6 @@ configure_file(enftun-setup ${CMAKE_CURRENT_BINARY_DIR}/enftun-setup COPYONLY)
 install(
   FILES ${CMAKE_CURRENT_BINARY_DIR}/enftun-setup
   DESTINATION ${CMAKE_INSTALL_BINDIR}
+  PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
```

Comment:
We should be using the `PROGRAMS` form of [install](https://cmake.org/cmake/help/latest/command/install.html#installing-files).  That will set the permissions automatically.

Reply:
Correct, much better, thanks